README.md

```diff
@@ -4,30 +4,32 @@ The Java software Discourse Network Analyzer (DNA) is a qualitative content anal
 
 - Download the latest [release](https://github.com/leifeld/dna/releases) of the software.
 
-- Check out the detailed [manual](https://github.com/leifeld/dna/releases/download/v2.0-beta.22/dna-manual.pdf) for more information, including installation instructions and information on network methods and rDNA.
+- Check out the detailed [manual](https://github.com/leifeld/dna/releases/download/v2.0-beta.23/dna-manual.pdf) for more information, including installation instructions and information on network methods and `rDNA`.
 
 - If you have questions or want to report bugs, please create an issue in the [issue tracker](https://github.com/leifeld/dna/issues).
 
 <br />
 
 ## rDNA. A Package to Control DNA from R
 
-This is the companion package to DNA. It integrates the Java software with the statistical computing environment R.
+This is the companion package to DNA. It integrates the Java software with the statistical computing environment `R`.
+
+You can install the most recent release using:
 
-You can install the most recent version using:
 ``` r
-# install.packages("devtools")
-devtools::install_github("leifeld/dna/rDNA", INSTALL_opts = "--no-multiarch")
+# install.packages("remotes")
+remotes::install_github("leifeld/dna/rDNA@*release", INSTALL_opts = "--no-multiarch")
 ```
-Please note that the most recent version requires compilation of the current jar file from the sources on GitHub, for example using the provided make file.
 
-The recommended version for most users is therefore the stable release version of rDNA, which can be used together with the jar file of the respective release (see [releases](https://github.com/leifeld/dna/releases)). You can install the stable release version of rDNA by inserting the path to the latest release version into the above command. For example, for the 2.0 beta 23 release, this would be:
+This is the recommended version for most users. Note that the package relies on `rJava`, which needs to be installed first. For details on the installation process on different operating systems, consult the chapter "Installation of DNA and rDNA" in the [manual](https://github.com/leifeld/dna/releases/download/v2.0-beta.23/dna-manual.pdf).
+
+You can run the same installation command above to update the software. However, the first time you run a new version of `rDNA`, invoke the command `dna_downloadJar()` to update `DNA` as well.
```

Comment:
Are we not risking here that somebody with an older rDNA installation gets a new, incompatible release version of the jar file?

Reply:
Also: change to version beta 24 here as well.

Reply:
If we leave it out they might end up with a new rDNA and an old jar though. Do you have an idea how to phrase it better? 

Or maybe we could query the release date of the jar file from R? Then we could write a simple test comparing it to `packageDescription("rDNA")$Date` and output a warning if either version is (much) older.

Reply:
Maybe "if you have the most recent release version of rDNA installed and would like to download the most recent release version of the DNA jar file, ..."? Or something similar. I think writing some script is overkill.

Reply:
I went a slightly different way. I think we should encourage people to update the software. So I tried to make it sound as straightforward as possible. Let me know what you think.